apps/passport-client/src/dispatch.ts

```diff
@@ -834,3 +848,32 @@ async function setKnownTicketTypesAndKeys(
     knownPublicKeys: keyMap
   });
 }
+
+function termsAgreed(state: AppState, update: ZuUpdate, version: number) {
```

Comment:
Oh, yes, these are actions and not events. This would have been the right variable name if we were using re-frame, which dispatches events and not actions!

Reply:
On second thoughts, the imperative `agreeToTerms` makes sense if what we're trying to do is tell `dispatch` to agree to the terms. But in this case we're telling it that the terms have already been agreed, and that it should update the state and close the modal. The correct name is something more like `closeTermsModal`.